src/Microsoft.DotNet.Darc/DarcLib/GitHubClient.cs

```diff
@@ -952,38 +922,23 @@ public async Task<IList<Review>> GetLatestPullRequestReviewsAsync(string pullReq
     {
         (string owner, string repo, int id) = ParsePullRequestUri(pullRequestUrl);
 
-        var reviews = await GetClient(owner, repo).Repository.PullRequest.Review.GetAll(owner, repo, id);
+        IGitHubClient client = GetClient(owner, repo);
+        var pullRequestReviewsUri = ApiUrls.PullRequestReviews(owner, repo, id);
+        string cacheKey = $"{owner}_{repo}_id";
```

Comment:
This one would have been nasty. Sounds like we could use a test?

```suggestion
        string cacheKey = $"{owner}_{repo}_{id}";
```

Reply:
I would also include more in the key, like `prState` or something to categorize what data we store.